sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetRebaseDatetimeSuite.scala

```diff
@@ -260,25 +260,25 @@ abstract class ParquetRebaseDatetimeSuite
                 }
 
                 withAllParquetReaders {
-                    // The file metadata indicates if it needs rebase or not, so we can always get
-                    // the correct result regardless of the "rebase mode" config.
+                  // The file metadata indicates if it needs rebase or not, so we can always get
+                  // the correct result regardless of the "rebase mode" config.
                   runInMode(inReadConf, Seq(LEGACY, CORRECTED, EXCEPTION)) { options =>
                     checkAnswer(
-                        spark.read.options(options).parquet(path).select($"ts".cast("string")),
-                        Seq.tabulate(N)(_ => Row(tsStr)))
+                      spark.read.options(options).parquet(path).select($"ts".cast("string")),
+                      Seq.tabulate(N)(_ => Row(tsStr)))
                   }
 
                   // Force to not rebase to prove the written datetime values are rebased
                   // and we will get wrong result if we don't rebase while reading.
                   withSQLConf("spark.test.forceNoRebase" -> "true") {
                     checkAnswer(
-                        spark.read.parquet(path).select($"ts".cast("string")),
-                        Seq.tabulate(N)(_ => Row(nonRebased)))
-                    }
-                  }
+                      spark.read.parquet(path).select($"ts".cast("string")),
+                      Seq.tabulate(N)(_ => Row(nonRebased)))
                   }
                 }
               }
+                }
+              }
```

Comment:
Is the indent correct for these brackets?

Reply:
Fixed.